integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonPatchTestUtils.java

```diff
@@ -198,4 +198,27 @@ public static boolean patchDomainResource(String domainUid, String domainNamespa
     V1Patch patch = new V1Patch(new String(patchStr));
     return patchDomainCustomResource(domainUid, domainNamespace, patch, V1Patch.PATCH_FORMAT_JSON_PATCH);
   }
+
+  /**
+   * Patch replicas at spec level.
+   *
+   * @param domainUid unique domain identifier
+   * @param domainNamespace the Kubernetes namespace where the domain is
+   * @param replicaCount the replica count to patch with
+   * @return true if successful, false otherwise
+   */
+  public static boolean patchReplicaCountAtSpecLevel(String domainUid, String domainNamespace, int replicaCount) {
```

Comment:
Should the code in oracle.weblogic.kubernetes.actions.impl.Operator#patchReplicas() be updated to use this new method?

Reply:
Operator#patchReplicas() is for operator and the one here is for domain CR. I can change the method name to patchDomainResourceWithNewReplicaCountAtSpecLevel().